patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -1380,12 +1380,25 @@
 +        return func_180642_a(world, pos, facing, hitX, hitY, hitZ, meta, placer);
 +    }
 +
++    /**
++     * Determines if another block can connect to this block
++     *
++     * @param world The current world
++     * @param pos The position of this block
++     * @param facing The side the connecting block is on
++     * @return True to allow another block to connect to this block
++     */
++    public boolean canBeConnectedTo(IBlockAccess world, BlockPos pos, EnumFacing facing)
```

Comment:
I would call it `canConnectTo` but it is not **what can this connect to** but **what can connect to this**.